ports/unix/mpconfigport.h

```diff
@@ -235,7 +235,7 @@ extern const struct _mp_obj_module_t mp_module_jni;
 #else
 #define MICROPY_PY_SOCKET_DEF
 #endif
-#if MICROPY_PY_USELECT_POSIX
+#if MICROPY_PY_USELECT_POSIX || MICROPY_PY_USELECT
```

Comment:
I'm wondering why this is needed.  `py/objmodule.c` already includes this module when `MICROPY_PY_USELECT` is enabled.  Isn't that enough?

Reply:
Yep, you're right. Reverted.